README.adoc

```diff
@@ -96,12 +96,15 @@ The plugin can be configured to inject Build Scans into any Gradle or Maven buil
 
 This feature works by installing scripts and Jars on each agent depending on the environment variable settings.
 
+The feature can be enabled by setting the `JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_INJECTION` environment variable.
+
 If the `JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_PLUGIN_VERSION` environment variable is present, injection is activated for Gradle builds.
-This causes an init script is installed in the user home of each connected agent.
-The init script is removed when the environment variable is removed, thus deactivating injection for Gradle builds.
+This causes an init script to be installed in the user home of each connected agent.
+The init script is removed when `JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_PLUGIN_VERSION` is set to `off`, thus deactivating injection for Gradle builds.
```

Comment:
This change (and the next) isn't necessary any more, right?

Reply:
well spotted, still relevant but the variable needs to be unset. I updated the doc accordingly